cl/citations/management/commands/import_citations_csv.py

```diff
@@ -58,12 +65,11 @@ def load_citations_file(options: dict) -> DataFrame | TextFileReader:
     return data
 
 
-def process_csv_data(
-    data: DataFrame | TextFileReader,
-) -> None:
+def process_csv_data(data: DataFrame | TextFileReader, options: dict) -> None:
```

Comment:
If the only thing you need is the delay parameter, it's much clearer just to pass that, instead of a big dict with a bunch of other stuff in it:

```suggestion
def process_csv_data(data: DataFrame | TextFileReader, delay_s: float) -> None:
```